src/components/login/register-user-with-captcha-view.vue

```diff
@@ -217,6 +217,13 @@ export default {
     },
     goToDocumentation() {
       window.location.href = this.HELP_LINK
+    },
+    downloadWallet() {
+      if (navigator.userAgent.toLowerCase().indexOf('iphone') > -1) {
+        window.location.href = 'http://itunes.apple.com/lb/app/id1659926348'
+      } else if (navigator.userAgent.toLowerCase().indexOf('android') > -1) {
+        window.location.href = 'http://play.google.com/store/apps/details?id=earth.hypha.wallet.hypha_wallet'
```

Comment:
@evgenibir Do not hardcode this links. Add them as env variables with default value set to http://play.google.com/store/apps/details?id=earth.hypha.wallet.hypha_wallet

Reply:
fixed